tt_metal/impl/buffers/buffer.hpp

```diff
@@ -197,6 +197,9 @@ class Buffer {
     }
 
     CoreCoord get_core_from_dev_page_id(uint32_t dev_page_id) const {
+        if(dev_page_id >= dev_page_to_core_mapping_.size()) {
+            std::cout << "WTF " << std::endl;
```

Comment:
😅 I'm guessing you probably didn't mean to leave this in

Reply:
lolol whoops, it has been removed :)